data/fixtures/scopes/php/disqualifyDelimiter.scope

```diff
@@ -0,0 +1,106 @@
+<?php
```

Comment:
what about this one?

Reply:
added. There is no node for `<` separate, but I think it's fine to just exclude that entire thing.